.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "firehose-source/src/main/proto"]
```

Comment:
We could just copy the proto files, but I think its more like a pattern I'm seeing on projects that need some sort of external IDL. But I'm ok on removing it if you prefer